### ISSUE TITLE
CanvasApp не работают в СберСалюте

### ISSUE THREAD

**Issue by and7ey** [sberdevices/salute-issues#33]
Делать два практически одинаковых смартапа, один - для Салюта, другой - для Бокса, выглядит странным.
Почему бы не добавить настройку в Canvas App, говорящую о том, этот смартап и без canvas-части он может полноценно?

**Comment by awinogradov** [comment]
CanvasApp будут работать в Салюте.

**Comment by VsevolodKiselev** [comment]
Срок реализации данного функционала 1 квартал 2021 года.  Можно будет создать несколько апликейшенов

**Comment by and7ey** [comment]
Зачем несколько апов? Ап должен быть один.
Просто для устройства без экрана вызовы на frontend endpoint делать не надо.

**Comment by awinogradov** [comment]
Это далеко не всегда верно. Сценарий чата и канваса может быть сильно разным в силу наличия того самого экрана.

**Comment by and7ey** [comment]
Конечно, может. Мое изначальное предложение для кейса, когда сценарий один. Если он один, то можно ставить галочку "ап будет работать хорошо и без экрана" и тогда использовать единый ап.

**Comment by subnetsRU** [comment]
Может всё же статус  "Deffered", а не "Closed" ?

**Comment by awinogradov** [comment]
В github issues нет такого понятния. Мы добавили колонку Defferd на борде тут https://github.com/sberdevices/salute-issues/projects/1

**Comment by and7ey** [comment]
а зачем вообще закрыли? создавать два идентичных апа - это странно (только представьте каково их поддерживать потом)

**Comment by subnetsRU** [comment]
>добавили колонку Defferd на борде

Ну я о ней и говорю. Прошу прощения, что не совсем корректно выразился.

**Comment by Genenko-Svetlana** [comment]
@subnetsRU можем перенести в выполненные? CanvasApp работают в СберСалюте.

**Comment by subnetsRU** [comment]
Ну строго говоря автор тикета не я. Да и тикет же не реализован. Как я понимаю его суть не в том, что CanvasApp не работает в Сбер Салют (тем более сейчас он там работает), а в том, чтобы под одним названием смартапа как то могли со существовать и chat app и canvas app.

**Comment by Genenko-Svetlana** [comment]
Срок реализации - 2 квартал 2021

**Comment by Genenko-Svetlana** [comment]
Возвращаем в работу

**Comment by Genenko-Svetlana** [comment]
Корректировка срока. Ориентировочно: 4 кв. 2021 - 1 кв. 2022.
Тикет временно закрываем и переносим в отложенные задачи.
